Presentation.Web/Tests/Helpers/SystemCatalogHelper.ts

```diff
@@ -88,6 +88,9 @@ class SystemCatalogHelper {
         return browser.wait(SystemCatalogHelper.pageObject.waitForKendoGrid(), SystemCatalogHelper.waitUpTo.twentySeconds);
     }
 
-
+    public static getValueFromSelect2(Selec2Element: string) {
```

Comment:
Mon ikke det er bedre du bruger Select2.getData direkte, for det parameter navn giver altså ingen mening.. er det et id, en class eller hvad er det?

Reply:
Bliver faktisk ikke brugt et levn fra en tidligere test jeg ikke brugte alligevel. ( fejl 40 )